tools/engine_tool/lib/src/commands/build_command.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:engine_build_configs/engine_build_configs.dart';
+import 'package:path/path.dart' as path;
+import 'package:process_runner/process_runner.dart';
+
+import '../build_utils.dart';
+
+import 'command.dart';
+
+const String _configFlag = 'config';
+
+// TODO(johnmccutchan): Should BuildConfig be BuilderConfig and GlobalBuild be BuildConfig?
+// TODO(johnmccutchan): List all available build targets and allow the user
+// to specify which one(s) we should build on the cli.
+// TODO(johnmccutchan): Can we show a progress indicator like 'running gn...'?
+
+/// The root 'build' command.
+final class BuildCommand extends CommandBase {
+  /// Constructs the 'query' command.
```

Comment:
Or ignore the lint with `// ignore: public_member_api_docs`

```suggestion
  /// Constructs the 'build' command.
```

Reply:
Done.